docs/reference/launchpad-text-markup.rst

```diff
@@ -105,11 +105,11 @@ The following pattern is used by Launchpad to detect bug references:
 
     LP: #<LP-Bug-Number>[, #<LP-Bug-Number>]... 
 
-This pattern is case invariant and the amount of whitespace can be variable, but if you place whitespace anywhere else the regular expression used by Launchpad might not parse the bug reference correctly!
+This pattern is case invariant, and the amount of white space can be variable, but if you place white space anywhere else, the regular expression used by Launchpad might not parse the bug reference correctly.
```

Comment:
I would actually like to propose avoiding the use of the term whitespace/white space here completely. Blank space or something similar should be a decently understandable alternative. Whitespace goes against the inclusive language guidelines.